### PR TITLE
wsd: disable ctrl+s when HideSaveOption is true

### PR DESCRIPTION
**Pull request by Rash419**
Change-Id: I190a03abcf80176cc2da77fd32190c683852b9a7

* Resolves: follow up of https://github.com/CollaboraOnline/online/commit/6b06f1d8fec1788c18c5755d7e5c2046c3407cbf
* Target version: master

### CODE REVIEW

**Review by eszkadev — COMMENTED** [review]
code looks ok, but let's double check with Ash if he doesn't have anything against

File: wsd/ClientSession.cpp [hunk 1]
```diff
@@ -690,6 +690,11 @@ bool ClientSession::_handleInput(const char *buffer, int length)
                                 << "] has no write permissions in Storage and cannot save.");
             sendTextFrameAndLogError("error: cmd=save kind=savefailed");
         }
+        else if(_wopiFileInfo && _wopiFileInfo->getHideSaveOption())
```
Comment (Ashod) [inline wsd/ClientSession.cpp:693]: Please use clang-format or manually format code to be consistent with C++ code in the project.

There is always a space after `if` tokens.

File: wsd/ClientSession.cpp [hunk 2]
```diff
@@ -690,6 +690,11 @@ bool ClientSession::_handleInput(const char *buffer, int length)
...
+        {
+            LOG_WRN("Session [" << getId() << "] on document [" << docBroker->getDocKey()
```
Comment (Ashod) [inline wsd/ClientSession.cpp:695]: I assume the UI will have blocked the save as well. So this warning is to signal that someone is trying to hack the UI?

**Review by Ashod — APPROVED** [review]
Thanks @Rash419. Looks good, but formatting can be improved.

File: wsd/ClientSession.cpp (hunk 1 above)
Comment (Rash419) [reply wsd/ClientSession.cpp:693]: yeah thanks for noticing, I just switched editor I will configure clang-format, which clang formatting style do we use ?

File: wsd/ClientSession.cpp (hunk 2 above)
Comment (Rash419) [reply wsd/ClientSession.cpp:695]: Yes

File: wsd/ClientSession.cpp (hunk 1 above)
Comment (Rash419) [reply wsd/ClientSession.cpp:693]: before this I was using vscode and it was falling back to its own style
 https://code.visualstudio.com/docs/cpp/cpp-ide

**Pull request merged**